packages/core/src/js/feedback/FeedbackWidget.styles.ts

```diff
@@ -63,30 +59,30 @@ const defaultStyles: FeedbackWidgetStyles = {
     marginRight: 10,
   },
   screenshotText: {
-    color: FOREGROUND_COLOR,
+    color: getTheme().foreground,
     fontSize: 16,
   },
   submitButton: {
-    backgroundColor: PURPLE,
+    backgroundColor: getTheme().accent,
     paddingVertical: 15,
     borderRadius: 5,
     alignItems: 'center',
     marginBottom: 10,
   },
   submitText: {
-    color: BACKGROUND_COLOR,
+    color: getTheme().background,
```

Comment:
Just a small oversight.
 
The submit button text should be in contrast with the button, not the widget background.

<img width="920" alt="Screenshot 2025-03-26 at 18 15 50" src="https://github.com/user-attachments/assets/97bddf20-b34e-425d-92b5-d5a6b3ba93b6" />

Reply:
Good catch Krystof 👍 
This escaped my attention. Updated with c831eeeac0886bed343c0755c7d8bed37cf8d984 and also updated the screenshots in the PR description to reflect the latest state.